parsers/main.js

```diff
@@ -9,4 +9,15 @@ module.exports.getParser = function(uniName, callback) {
 		var parser = require('./' + uniName + '.js');
 		return callback(null, parser.convertToCal);		
 	});		
+}
+
+module.exports.getParser2 = function(uniName, callback) {
```

Comment:
completely replace existing parser with this, we don't want the old stuff any more